elements/pfe-tabs/BaseTabs.ts

```diff
@@ -199,34 +216,40 @@ export abstract class BaseTabs extends LitElement {
     }
   }
 
-  protected async _onSlotChange(): Promise<void> {
-    this._tabs.forEach((tab, index) => {
+  protected _onSlotChange(): void {
+    this.allTabs = this._tabs;
+    this.#firstLastClasses();
+    this.allPanels = this._panels;
+    this.#updateAccessibility();
+  }
+
+
+  #updateAccessibility(): void {
+    this.allTabs.forEach((tab: BaseTab, index: number) => {
+      const panel = this.allPanels[index];
+      panel.setAriaLabelledBy(tab.id);
+      tab.setAriaControls(panel.id);
+    });
+  }
+
+  #firstLastClasses() {
+    this.allTabs.forEach((tab, index) => {
       if (index === 0) {
         tab.classList.add('first');
       } else {
         tab.classList.remove('first');
         tab.classList.remove('last');
       }
-      if (index === this._tabs.length - 1) {
+      if (index === this.length - 1) {
         tab.classList.add('last');
       }
     });
-    await this.updateComplete;
-    this.#updateAccessibility();
-  }
-
-  #updateAccessibility(): void {
-    this._allTabs().forEach((tab: BaseTab, index: number) => {
-      const panel = this._allPanels()[index];
-      panel.setAriaLabelledBy(tab.id);
-      tab.setAriaControls(panel.id);
-    });
   }
 
   @bound
   private _tabExpandEventHandler(event: Event): void {
```

Comment:
```suggestion
  #onTabExpand(event: Event): void {
```

### this binding
doesn't need the decorator, since `addEventListener` is called on the element instance (L59). same for `_onKeyDownHandler`. Similarly, anything bound in a render template listener will get auto-bound to the element instance, so `@bouunud _scrollLeft` can be `#scrollLeft`, etc
for handleScrollButtons, the listener on tablist should probably be removed in disconnectedcallback. We could remove the dependency on `@bound` by making it a class field. I'm not yet solid on which of those is really preferable, but my feeling now is that if that's the last instanced of `@bound` in the file, then better to remove it.
```ts
#onScroll = () => {
  clearTimeout(...)
  ...
}
```

### Methods Naming
let's use `onEvent` instead of `eventHandler` - it's fewer bytes and more consistent